proposals/csharp-12.0/collection-expressions.md

```diff
@@ -356,13 +356,17 @@ var z = Extensions.AsImmutableArray([3]); // ok
 
 [*Better conversion from expression*](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/expressions.md#11644-better-conversion-from-expression) is updated to prefer certain target types in collection expression conversions.
 
+In the updated rules:
+* A *span_type* is `System.Span<T>` or `System.ReadOnlySpan<T>`.
+* An *array_or_array_interface_type* is an *array type* or one of the following *interface types* from `System.Collections.Generic` implemented by an *array type*: `IEnumerable<T>`, `IReadOnlyCollection<T>`, `IReadOnlyList<T>`, `ICollection<T>`, `IList<T>`.
```

Comment:
i believe the spec already states this elsewhere (i think in the conversions section).  Let's link there instead?  That way if that list updates, it applies here?

Reply:
I will leave the interface types here explicitly for now since the two sections are otherwise unrelated.